source/community/release_notes/quince.rst

```diff
@@ -15,26 +13,154 @@ These are the release notes for the Quince release, the 17th community release o
+  * The constructor kwarg bucket is no longer accepted. Instead, use bucket_name.
+  * define default_acl value explicitly in constructor kwarg e.g default_acl: public-read. in previous versions django-storages provides default value as public-read but now it is none. So it's important to mention it explicitly as per your use case.
+* For Django 4.2 CSRF_TRUSTED_ORIGINS must include scheme. Update all urls there with schemes. e.g .edx.org changes to ``https://*.edx.org``.
+* Docker requirements have changed: They are now: Docker v24.0.5+, with BuildKit 0.11+. 
 
 Learner Experiences
 *******************
 
+* `feat: fetch program subscription details <https://github.com/openedx/edx-platform/pull/32023>`__
+
 
 Instructor Experiences
 **********************
 
+* `Copy and Paste For Course Components in Studio <https://openedx.org/blog/introducing-copy-paste-for-course-components/>`__
 
 Administrators & Operators
 **************************
 
+New Learner Home Page 
+==================================
+
+No longer experimental, the new Learner Home has many of the same features as the old learner dashboard, with some extended functionality and 
+performance enhancements.   
+
+* The Learner Home is now built with Paragon, the Open edX design pattern library. It is accessible and easy to style with brand colors. 
+* Course cards show the course thumbnail, information about the course, and the ability to upgrade to a paid track or view/begin a course (if applicable). Further course actions (e.g. unenroll, email opt-out settings, and social media share) have been moved to the menu/triple dot icon on the course card.
+* Clicking the “Refine” button opens options to filter by course status or sort either by most recent enrollment (default) or title.
+
+.. image:: /_images/community/release_notes/palm/new_learner_home_filtering.png
+    :alt: The Refine pop-up with Course State and Sort options
+
+* All of a user’s courses are fetched on page load. To make the page manageable, we paginate that list, showing 25 courses at a time. To view other courses, a user should page through their list of courses using the pagination controls at the bottom of the course list.
+
+.. image:: /_images/community/release_notes/palm/new_learner_home_pagination.png
+    :alt: Next, previous and page number buttons appear below the list of courses
+
+* Site staff can now masquerade as users on the platform by typing a username or email in the “View as” box and hitting submit. This is designed to be “view only” so change actions (e.g. enroll, unenroll, selecting a session) are blocked in this view.
+* There is a complete list of changes in the `Open edX wiki <https://openedx.atlassian.net/wiki/spaces/OEPM/blog/2022/11/21/3584589831/2U+New+Learner+Home+Page#Comparison-with-old-experience>`_. 
+* `feat: feature flag to disable Advanced Settings <https://github.com/openedx/edx-platform/pull/32015>`__
+
+
```

Comment:
New in Quince is the in-unit discussion sidebar; I believe it's important enough to warrant a subsection here, at the same level as "New Learner Home Page".  I added a few details in the PR to add support for it in tutor-mfe. Feel free to copy anything from that:

https://github.com/overhangio/tutor-mfe/pull/176

Reply:
Also, there's a new MFE that gets deployed as part of [tutor-credentials](https://github.com/overhangio/tutor-credentials): [Learner Record](https://github.com/openedx/frontend-app-learner-record/).  It might warrant a blurb here as well.  Some basic info was added the tutor-credentials README via [the corresponding PR](https://github.com/overhangio/tutor-credentials/pull/17/files).

Alternatively, if Credentials itself wasn't mentioned in Palm, it might be worth calling it out as a whole instead: it's now an officially (Edly) supported Tutor plugin.